### PR TITLE
HoC 2022 - Hoc.com updates for soon-hoc - Oct launch

### PR DESCRIPTION
**Pull request by kelbyhawn**
Updates on https://hourofcode.com for Hour of Code 2022 for `soon-hoc`.
- Tested each banner in different hoc_modes: `pre-hoc`, `soon-hoc`, `actual-hoc`, `post-hoc`, `false`

**Related PRs:** https://github.com/code-dot-org/code-dot-org/pull/48292 • https://github.com/code-dot-org/code-dot-org/pull/48187

**Asana tasks:** [Homepage](https://app.asana.com/0/1202764401306810/1202772113384569/f) • [Activities page](https://app.asana.com/0/1202764401306810/1202772113384590/f)

## HoC.com Homepage - https://hourofcode.com/us
- Update hero banner

**Desktop:**
Desktop

**Tablet:**
Tablet

**Mobile:**
Mobile

## Activities page - https://hourofcode.com/us/learn
- Update hero banner

**Desktop:**
Desktop

**Tablet:**
Tablet

**Mobile:**
Screen Shot 2022-09-30 at 2 44 56 PM

### CODE REVIEW

File: pegasus/sites.v3/hourofcode.com/public/index.haml [hunk 1]
```diff
@@ -34,72 +35,23 @@ social:
...
-      &nbsp;
-
+#top
+  #fullwidth
     = view :header
-
-    .container
-      .row
+    %section.banner.homepage
+      .wrapper
```
Comment (megcrenshaw) [inline pegasus/sites.v3/hourofcode.com/public/index.haml:42]: So glad this styling is moving out of haml!

File: pegasus/sites.v3/hourofcode.com/public/index.haml [hunk 2]
```diff
@@ -34,72 +35,23 @@ social:
...
         - if ["pre-hoc", "soon-hoc", "actual-hoc", "post-hoc"].include?(hoc_mode)
-          %div{style: "color: black; display: flex"}
```
Comment (megcrenshaw) [inline pegasus/sites.v3/hourofcode.com/public/index.haml:68]: So glad this styling is moving out of haml!

**Review by megcrenshaw — APPROVED** [review]
Looks great to me! And I'm thrilled about this styling being in .scss now 🥳

File: pegasus/sites.v3/hourofcode.com/public/css/hoc-banner.scss [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    @media screen and (max-width: $width-sm) {
+      padding: 1em 1em 3em;
+      width: 88%;
+    }
+  }
+
+  a {
+    color: $darkest_gray;
+    text-decoration: underline;
+    font-family: $gotham-bold;
```
Comment (kelbyhawn) [inline pegasus/sites.v3/hourofcode.com/public/css/hoc-banner.scss:98]: @megcrenshaw I just replaced `color:` and `font-family:` values w/ scss variables 🙌  Probs doesn't need another review, but wanted to point this out.

Comment (megcrenshaw) [reply pegasus/sites.v3/hourofcode.com/public/css/hoc-banner.scss:98]: Ah, that's great! Good call 👍

**Review by davidsbailey — APPROVED** [review]
looks good to me!

**Pull request merged**